distributed/cli/dworker.py

```diff
@@ -41,16 +41,19 @@ def main(center, host, port, nthreads, nprocs, no_nanny):
     if not nthreads:
         nthreads = _ncores // nprocs
 
-    if nprocs > 1 and port != 0:
-        raise ValueError("Can not specify a port when using multiple processes")
-
```

Comment:
This check still seems valid to me.  I should add more tests around the command line interface.